websites/N/Necord/metadata.json

```
@@ -0,0 +1,54 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "someone",
+		"id": "235413185639874561"
+	},
+	"service": "Necord",
+	"description": {
+		"en": "A module for creating Discord bots using NestJS, based on Discord.js"
+	},
+	"url": "necord.org",
```

Site uses `www` subdomain.
```suggestion
	"url": [
		"necord.org",
		"www.necord.org"
	],
```